process_testcase.go

```diff
@@ -412,6 +412,9 @@ func (v *Venom) printTestStepResult(tc *TestCase, ts *TestStepResult, tsIn *Test
 				v.Println(" %s", Gray(StatusSkip))
 			} else {
 				v.Println(" %s", Green(StatusPass))
+				if ts.Retries > 1 {
+					v.Println(" \t\t  %s", Green(fmt.Sprintf("It's a success after %d attempts", ts.Retries)))
+				}
```

Comment:
```suggestion
				if ts.Retries == 0 {
				        v.Println(" %s", Green(StatusPass))
				} else {
					v.Println(" %s (after %d attempts)", Green(StatusPass),  ts.Retries)
				}
```

Reply:
Hello,
Thank you for your feedback, it's been taken into account